modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/WebtoolsAnalyticsRuleListBuilder.php

```diff
@@ -31,11 +39,28 @@ public function buildRow(EntityInterface $entity): array {
     if (!$entity instanceof WebtoolsAnalyticsRuleInterface) {
       throw new \InvalidArgumentException('Only Webtools Analytics rules can be listed.');
     }
-    $row['section'] = $entity->getSection();
-    $row['match_on_site_default_language'] = $entity->matchOnSiteDefaultLanguage() ? $this->t('Yes') : $this->t('No');
-    $row['id'] = $entity->getRegex();
+    $row['label'] = $entity->getSection();
+    $row['regexp'] = [
+      '#plain_text' => $entity->getRegex(),
+    ];
+    $row['match_on_site_default_language'] = [
+      '#plain_text' => $entity->matchOnSiteDefaultLanguage() ? $this->t('Yes') : $this->t('No'),
+    ];
 
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildForm($form, $form_state);
+    $form['#prefix'] = '<p>' . t('This table provides a drag-and-drop interface for managing matching priority of <em>webtools analytics rules</em> for each page from top to bottom.') . '</p>';
```

Comment:
Let's go for something like:
`The analytics rules are processed from top to bottom. The order can be re-arranged through drag-and-drop.`

Reply:
Also use `$this->t()`.